src/duty/attestation/attestation.metrics.ts

```diff
@@ -57,81 +57,87 @@ export class AttestationMetrics {
   private async perfectAttestationsLastEpoch() {
     const result = await this.storage.getValidatorCountWithPerfectAttestationsLastEpoch(this.processedEpoch);
     this.operators.forEach((operator) => {
-      const operatorResult = result.find((a) => a.val_nos_name == operator.name);
-      this.prometheus.validatorsCountPerfectAttestation.set({ nos_name: operator.name }, operatorResult ? operatorResult.amount : 0);
+      const operatorResult = result.find((a) => a.val_nos_id != null && +a.val_nos_id == operator.index);
```

Comment:
Can you elaborate on what `+a` stands for?

Reply:
Cast `String` to `Number`